mev_inspect/schemas/liquidations.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel
+from mev_inspect.schemas.classified_traces import Protocol
+
+
+class Liquidation(BaseModel):
+    liquidated_user: str
+    liquidator_user: str
+    # collateral
+    collateral_token_address: str
+    # collateral_amount: int
+    # debtToCover
+    debt_token_address: str
+    # purchaseAmount
```

Comment:
i don't think these should be here since they're aave specific

if the field names are unclear such that they need a comment, we should change the names to be clearer

also think this connection is made clear enough in the `get_inspections` itself where you're doing stuff like
```
debt_purchase_amount=trace.inputs["_purchaseAmount"]
```

Reply:
Got it